src/instructlab/data/generator/utils.py

```diff
@@ -0,0 +1,228 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
?

Reply:
sorry, accepted the wrong changes when rebasing :)